lib/active_merchant/billing/gateways/epay.rb

```diff
@@ -186,7 +186,7 @@ def do_authorize(params)
         # Authorize gives the response back by redirecting with the values in
         # the URL query
         if location = response['Location']
-          query = CGI::parse(URI.parse(location.gsub(' ', '%20')).query)
+          query = CGI::parse(URI.parse(location.gsub(' ', '%20').gsub('<', '%3C').gsub('>', '%3E')).query)
```

Comment:
`gsub` takes a map as its second parameter IIRC, so you should be able to do this with a single call rather than chaining them together